### ISSUE TITLE
`no-invalid-position-at-import-rule` doesn't ignore `@use` statements

### ISSUE THREAD

**Issue by elliotleelewis** [kristerkari/stylelint-scss#524]
When running stylelint with `no-invalid-position-at-import-rule` enabled, the following fails:

```scss
@use 'sass:selector';

@import './styles/icons';
```

![image](https://user-images.githubusercontent.com/17044667/118031971-1f9db580-b335-11eb-9228-519a5fc78741.png)

But after switching them the SCSS compile fails, stating that `@use rules must be written before any other rules.`:

![image](https://user-images.githubusercontent.com/17044667/118031618-bddd4b80-b334-11eb-8f4d-81a7659a5d22.png)

**Comment by kristerkari** [comment]
Thanks for reporting this!

**Comment by kristerkari** [comment]
I somehow missed that the rule in the title was from stylelint core, so there is nothing that I can do about it. I guess that the rule can be disabled if it does not work with scss.

**Comment by elliotleelewis** [comment]
@kristerkari We probably need to create a replacement rule in `stylelint-scss` (`scss/no-invalid-position-at-import-rule`), as it looks like the Stylelint team aren't going to fix it. 😢 

https://github.com/stylelint/stylelint/issues/5322

I guess that as it only breaks with the SCSS custom syntax, this is probably the correct solution anyway, it's not really an issue with Stylelint itself.
